stack-packs/pack.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Pack {
```

Comment:
can this be inlined in `index.d.ts` rather than importing and reexporting it there?